src/sidebar/components/top-bar.js

```diff
@@ -52,16 +69,21 @@ function TopBar({
     store => store.getState().sidebarPanels.activePanelName
   );
 
+  const searchQueryProp = {};
+  if (filterQuery) {
+    searchQueryProp.query = filterQuery;
+  }
+
   /**
    * Open the help panel, or, if a service callback is configured to handle
    * help requests, fire a relevant event instead
    */
   const requestHelp = () => {
-    const service = serviceConfig(settings) || {};
-    if (service.onHelpRequestProvided) {
+    const service = serviceConfig(settings);
+    if (service && service.onHelpRequestProvided) {
       bridge.call(bridgeEvents.HELP_REQUESTED);
     } else {
-      togglePanelFn(uiConstants.PANEL_HELP);
+      togglePanelFn(uiConstants.PANEL_HELP, false);
```

Comment:
Passing the value `false` here will hide the sidebar panel every time it is called. Omitting the second parameter toggles the panel, which is the intent.

Reply:
bah! that was a debug thing I forgot to remove, good catch!